test/Microsoft.Framework.Runtime.FunctionalTests/ProjectFileGlobbing/ProjectFilesTests.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.IO;
+using Microsoft.Framework.Runtime.FileGlobbing;
+
+namespace Microsoft.Framework.Runtime.FunctionalTests.ProjectFileGlobbing
+{
+    public class ProjectFileGlobbingFacts : ProjectFilesCollectionTests
```

Comment:
Same set of tests in `ProjectFilesCollectionFacts` will be run again here but through a `Project` instance. Intend to test the integration of `Project` and `ProjectFilesCollection`.